doc/devdocs/readme.md

```diff
@@ -50,8 +50,23 @@ Various tools used by PowerToys. Includes the Visual Studio 2019 project templat
 
 ### Build Prerequisites
 
+1. install the following:
```

Comment:
This step says "install the following" but then step 2 walks the user through install. Doesn't make a lot of sense. Instead, I recommend deleting this step and letting the heading above speak for it. Then add, perhaps, `### Step-by-step instructions` for the actual steps below.

Full example:

### Requirements
- Visual Studio Community 2019 or newer
- Windows 10 November 2019 (version 1909) or newer

### Step-by-step
1. Do something
2. Do something else

Reply:
adjusted